services/ingest.json

```diff
@@ -11,7 +11,7 @@
     {
       "groupId": "org.jenkins-ci.plugins",
       "artifactId": "configuration-as-code",
-      "url": "http://repo.jenkins-ci.org/incrementals/io/jenkins/configuration-as-code/0.9-alpha-rc292.ee88abe834b2/configuration-as-code-0.9-alpha-rc292.ee88abe834b2.hpi",
+      "url": "http://repo.jenkins-ci.org/incrementals/io/jenkins/configuration-as-code/0.9-alpha-rc304.ab2fad387d16/configuration-as-code-0.9-alpha-rc304.ab2fad387d16.hpi",
```

Comment:
@mandie722 FYI, this hpi was built the so-called [JEP 305, aka _Incrementals_](https://github.com/jenkinsci/jep/tree/master/jep/305).